appinfo/info.xml

```diff
@@ -44,6 +44,7 @@ More information is available in the External sites documentation.]]></descripti
 	<repair-steps>
 		<post-migration>
 			<step>OCA\External\Migration\CopyDefaultIcons</step>
+			<step>OCA\External\Migration\JWTTokenPrivateKeySensitive</step>
```

Comment:
So you're going to run this on each upgrade. Don't you wanna run it only once?
I did that in a migration step in a bunch of apps.

Reply:
Abusing the database migrations to do this feels wrong to me.
Executing this repair step is extremely cheap, so I think this is fine.